packages/frontend/mui.types.d.ts

```diff
@@ -32,3 +32,5 @@ declare module "@mui/material/Typography" {
     label: true;
   }
 }
+
+export type RetroPaletteMode = "highContrast" | PaletteMode;
```

Comment:
Added a mode which do not exist yet as preparation for more themes

Reply:
```suggestion
export type RetroPaletteMode = "light" | "dark";
```

using `| PaletteMode` results in an `any` type for some reason. Let's omit this and just define the enum ourselves.

Reply:
Type resulted in any due to a missing import statement for PaletteMode. In current implmentation the type does not exist anymore